tests/unit/mathematics/linalg/LinalgBackendViennaCL_unittest.cc

```diff
@@ -5,7 +5,7 @@
 #include <thread>
 
 #ifdef HAVE_VIENNACL
-#include <shogun/mathematics/linalg/LinalgBackendViennaCL.h>
+#include <shogun/mathematics/linalg/LinalgBackendViennacl.h>
```

Comment:
umm.. why not merge this PR instead as it fixes the issue? It will have to wait till another 8 hours before I could rebase and merge this PR.. although there is no hurry..